test/commands/commerce/extension/points/list.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { strict as assert } from 'assert';
+import { stubInterface } from '@salesforce/ts-sinon';
+import { IConfig } from '@oclif/config';
+import { $$ } from '@salesforce/command/lib/test';
+import sinon from 'sinon';
+import { QueryResult } from '@mshanemc/plugin-helpers/dist/typeDefs';
+import { getEPN } from '../../../../../src/commands/commerce/extension/points/list';
+import * as forceOrgSoqlExports from '../../../../../src/lib/utils/sfdx/forceDataSoql';
+import { Result } from '../../../../../src/lib/utils/jsonUtils';
+
+describe('Test EPN list command', () => {
+    const config = stubInterface<IConfig>($$.SANDBOX, {});
+    const QUERY_GET_EPN =
+        "SELECT Value, IsDefaultValue, IsActive FROM PicklistValueInfo WHERE EntityParticle.DurableId = 'RegisteredExternalService.ExtensionPointName'";
+    const epnList = new getEPN([], config);
+
+    after(() => {
+        sinon.restore();
+    });
+
+    it('test #1', async () => {
```

Comment:
If you can name the tests based on what they are testing about, it will be easier to read and add to it later.